riotfile.py

```diff
@@ -516,11 +516,19 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                         "django": [
                             ">=2.1,<2.2",
                             ">=2.2,<2.3",
+                        ],
+                    },
+                ),
+                Venv(
+                    pys=select_pys(min_version="3.6"),
+                    pkgs={
+                        "django": [
                             "~=3.0",
                             "~=3.0.0",
                             "~=3.1.0",
                             "~=3.2.0",
                         ],
+                        "channels": latest,
```

Comment:
should we be testing on explicit versions of channels?

does our change/patching work on all versions of channels or only latest?

Reply:
This issue was flagged in the latest version of channels (v3.0.4) so that's why it's pinned to latest. Looking through previous versions this fix should also work for v2.X but it would also slow down ci

Reply:
I don't have a strong opinion on this one. We can only test against `latest` or add `"channels": ["~=2.0", "~=2.1",  "~=2.2", "~=2.3",  "~=2.4", "~=3.0",  latest]`

Reply:
doing each minor of channels is probably not worth it, but latest of each major would

also, `~=2.0`/`~=2.1`/`~=2.2`/etc all will give you the same version. `~=` will treat the last provided number as a wildcard. e.g. `~=2.1` == `>=2.1,<3.0`

`"channels": ["~=2.4", "~=3.0", latest]` is likely all we'd need.